tools/proofers/preview.js

```diff
@@ -689,6 +704,49 @@ $(function () {
             }
         }
 
+        // check that math delimiters \[ \], \( \) are correctly matched
+        function checkMath() {
+            var mathRe = /\\\[|\\\]|\\\(|\\\)/g;
+            var result;
+            var tag, openTag = null;
+            var openStart = 0;
+            while (true) {
+                result = mathRe.exec(txt);
+                if (null === result) {
+                    // if there is a tag open mark it as an error
+                    if (openTag) {
+                        reportIssue(openStart, 2, "noEndTag");
+                    }
+                    break;
+                }
+                tag = result[0].charAt(1);
+                // if no open tag and ( or [ set open else error
+                if (!openTag) {
+                    if ((tag === '(') || (tag === '[')) {
+                        openTag = tag;
+                        openStart = result.index;
+                    } else {
+                        reportIssue(result.index, 2, "noStartTag");
```

Comment:
It would be nice if we had an enum type for this. Something like:
`
var ISSUES = {
  /* ... */
  NO_START_TAG: 'noStartTag',
};
`

Reply:
Yes, I'll look into this and perhaps make another branch to address it.